src/components/Menu/MenuItem.tsx

```diff
@@ -14,12 +14,12 @@ export const useStyles = makeStyles(
       cursor: 'pointer',
       display: 'flex',
       fontFamily: theme.typography.fontFamily,
-      fontSize: theme.typography.body1.fontSize,
+      fontSize: theme.pxToRem(14),
```

Comment:
Curious, is there a move away from a fixed list of styles like `body1`?

Reply:
Tony and I decided awhile back to start using `pxToRem` over the fixed list but I can't remember the exact reason as it's been so long ago. We have a mix of both in Chroma now. I realize this is a terrible answer to your question.